src/wallet/rpcwallet.cpp

```diff
@@ -175,6 +175,36 @@ static std::string LabelFromValue(const UniValue& value)
     return label;
 }
 
+static void SetFeeEstimateMode(CWallet* const pwallet, CCoinControl& cc, const UniValue& estimate_mode, const UniValue& estimate_param)
+{
+    if (!estimate_mode.isNull()) {
+        if (!FeeModeFromString(estimate_mode.get_str(), cc.m_fee_mode)) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid estimate_mode parameter");
+        }
+    }
+
+    if (cc.m_fee_mode == FeeEstimateMode::BTC_KB || cc.m_fee_mode == FeeEstimateMode::SAT_B) {
+        if (estimate_param.isNull()) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Selected estimate_mode requires a confirmation target");
+        }
+
+        CAmount fee_rate = AmountFromValue(estimate_param);
+
+        if (cc.m_fee_mode == FeeEstimateMode::SAT_B) {
+            fee_rate /= 100000; // 1 sat / B = 0.00001 BTC / kB
+        }
+        cc.m_feerate = CFeeRate(fee_rate);
+        // if feerate < 0.1 sat/b, throw
+        if (fee_rate < 100) {
```

Comment:
This shouldn't be hard-coded

Reply:
Turned into constant. Also did same for the conversion a few lines above.